### PR TITLE
fix: automatically collapse series UI when expanding another

### PR DESCRIPTION
**Pull request by owlas**
### Pre requirements:
- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: 

### Scope of the changes (select all that apply):
- [ ] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

File: packages/frontend/src/components/ChartConfigPanel/Series/BasicSeriesConfiguration.tsx [hunk 1]
```diff
@@ -27,6 +27,9 @@ const BasicSeriesConfiguration: FC<BasicSeriesConfigurationProps> = ({
     getSeriesColor,
     updateSingleSeries,
 }) => {
+    const [openSeriesId, setOpenSeriesId] = React.useState<string | undefined>(
+        undefined,
```
Comment (rephus) [inline packages/frontend/src/components/ChartConfigPanel/Series/BasicSeriesConfiguration.tsx:31]: is this not the same as `React.useState<string | undefined>()`

File: packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx [hunk 2]
```diff
@@ -78,7 +78,7 @@ const ChartConfigTabs: FC = () => {
                 }),
                 {},
             );
-    }, [dirtyEchartsConfig]);
+    }, [dirtyEchartsConfig, orgData]);
```
Comment (rephus) [inline packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx:81]: is this orgData needed here ?

**Review by rephus — APPROVED** [review]
LGTM, I only have a couple of questions

File: packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx (hunk 2 above)
Comment (owlas) [reply packages/frontend/src/components/ChartConfigPanel/ChartConfigTabs.tsx:81]: Yes! Inside the reduce there's call to `orgData` -> `orgData?.chartColors ...` L76

**Comment by owlas** [comment]
@rephus I've re-requested a review because I didn't need one of the implementations of `isOpen` and `toggleIsOpen` because I noticed that if `isCollapsible===false` then those properties aren't used. 

Seems the `SingleMetricConfiguration` component serves two quite different use cases.

**Review by rephus — APPROVED** [review]

**Pull request merged**